Tweak.xm

```diff
@@ -30,9 +39,12 @@ static BOOL shouldShowPicker = NO;
 {
 	if (shouldShowPicker)
 	{
-		//WiPi already active
+		//WiPi already active, It does not work well on lock screen.
 		return;
 	}
+    
+    //It does not work well on lock screen on iOS10.
+    if (kCFCoreFoundationVersionNumber >= 1348.00 && [[objc_getClass("SBUserAgent") sharedUserAgent] deviceIsLocked]) return;
```

Comment:
You should use `%c("Class")` instead of `objc_getClass`. Logos will cache the class so you never have to look it up via the runtime more than once